manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -202,7 +202,7 @@ spec:
             path: /var/run/istio-cni
         - name: cni-netns-dir
           hostPath:
-            path: {{ .Values.cniNetnsDir | default "/var/run/netns" }}
```

Comment:
can someone rename this to `cni.netnsDir`?

Reply:
this is to make consistent with https://github.com/istio/istio/blob/b71a015135aacaf8dee6e38fcda2f5b43566f7d6/manifests/charts/istio-cni/values.yaml#L25

Reply:
BTW, I'm OK to rename variable ;)